tests/test_resources/kadet_functions/add_team_label/__init__.py

```diff
@@ -0,0 +1,71 @@
+import os
+import json
+from kapitan.resources import jinja2_render_file, yaml_load
+from kapitan.inputs import kadet
+import glob
+import yaml
+import time
+
+inventory = kadet.inventory()
+
+
+def set_team_name(obj, team_name):
+    obj.root.metadata["labels"]["team_name"] = team_name
+    return obj
+
+
+def set_namespace(obj, namespace):
+    obj.root.metadata["namespace"] = namespace
+    return obj
+
+
+def main(kadet_params):
+    team_name = kadet_params.get("team_name", "no-owner")
+    if "namespace" in kadet_params:
+        namespace = kadet_params.get("namespace")
+    else:
+        raise ValueError("'namespace' key not found in 'kadet_params'")
+
+    if "input_paths" in kadet_params:
+        input_paths = kadet_params.get("input_paths")
+    else:
+        raise ValueError("'input_paths' key not found in 'kadet_params'")
+
+    # get path where files have been compiled on this run
+    target_name = inventory.parameters.kapitan.vars.target
+    compile_path = kadet_params.get("compile_path")
+    root_path = ""
+    for s in compile_path.split("/"):
```

Comment:
Don't hardcode path delimiters. This might not work the same on all operating systems in the future. 
https://docs.python.org/2/library/os.path.html#os.path.split use os.path.split() or smth similar to get all path components. 
Maybe os.path.basename (?).